HW1/P3/P3.py

```diff
@@ -0,0 +1,41 @@
+import seaborn as sns
+sns.set_context('poster', font_scale=1.25)
+import findspark as fs
+fs.init()
+import pyspark as ps
+import multiprocessing as mp
+import string
+
+### Setup ###
+config = ps.SparkConf()
+config = config.setAppName('anagram_solver')
+sc = ps.SparkContext(conf=config)
+
+### Parsing the data efficiently! ###
+wlist = sc.textFile('EOWL_words.txt', use_unicode=True)
+
+def get_anagrams_key(input_str):
+    alphabet = string.ascii_lowercase
+    key = [0 for z in range(len(alphabet))]
```

Comment:
Interesting way of doing this.  It would use less space, and may be simpler just use sorted letter sequences instead of arrays of character counts that are length 26.